### ISSUE TITLE
 DNLA Japanese support

### ISSUE THREAD

**Issue by tesshucom** [tesshucom/jpsonic#305]
Implemented after #306.

- [x] Japanese localization of Title used in processor
##### Fixed element sorting to Japanese sort
 - [x] Album
 - [x] Artist
 - [x] Genre
 - [x] Folder
 - [x] Playlist
 - [x] RecentAlbum

**Comment by tesshucom** [comment]
Asynchronous communication is performed every 30 counts.
Therefore, it is necessary to solve the sort in the Dao hierarchy.

**Comment by tesshucom** [comment]
Album / artist / folder improved by #308.

Check the operation of Genre / Playlist / RecentAlbum.
If there is no big problem, I will postpone it.

**Comment by data-struct** [comment]
Changed DNLA sorting specifications as follows.
As with the Web, specifications change depending on the display hierarchy.

Album(id3) : eng/jp alphabetical fixed.

Artist(id3) : eng/jp alphabetical
Artist/Album(id3) : Depends on settings. Year or  eng/jp alphabetical.

Artist(folder) : eng/jp alphabetical fixed.
Artist/Album(folder) : Depends on settings. Year or  eng/jp alphabetical.

Added support for sequential number sorting.

**Comment by data-struct** [comment]
Genre : Same as Web. Count order. Do not modify this time.
Playlist : Simple name sorting. `Need to be shared`.
RecentAlbum : Web and results are different. `Need to be shared`.

**Comment by tesshucom** [comment]
RecentAlbum often displays differently on the Web and DNLA.
(It's not completely different, it's a little different.)
There's no big difference in the code and I don't know why.


DNLA improvements will continue.
Therefore, RecentAlbum issue issues a new issue and this PR is closed.
